config/roles.yml

```diff
@@ -45,7 +45,7 @@ alerting_ack_alerts:
     - 'cluster:admin/opendistro/alerting/alerts/*'
     - 'cluster:admin/opendistro/alerting/chained_alerts/*'
     - 'cluster:admin/opendistro/alerting/workflow_alerts/*'
-    - 'cluster:admin/opensearch/alerting/comments/search'
+    - 'cluster:admin/opensearch/alerting/comments/*'
```

Comment:
Given that the role name is ``alerting_ack_alerts`` , does this require to have full access to ``comments``?

Reply:
The feedback came up that one who can acknowledge alerts should be able to add comments to that alert as well, as opposed to only be able to read the alert's comments. Giving the alerting_ack_alerts role full alerting comments access is something I discussed and confirmed with PM Joshua Bright (@jdbright).